### ISSUE TITLE
Getting ExecuteError when trying to loop CalculateAccessibilityMatrix

### ISSUE THREAD

**Issue by vancouvertransportation** [Esri/public-transit-tools#169]
I'm trying to loop the tool through various iterators such as cut-off time, weight field etc. The script will get through a single iteration and return with an execution error. Does anyone know how to remedy this issue? 

The specific error I'm getting is: 
---------------------------------------------------------------------------
ExecuteError                              Traceback (most recent call last)
In  [12]:
Line 22:    solvetransitodcm()

In  [12]:
Line 21:    Precalculate_Network_Locations=True)

File T:\13-1400-10\TPL - GIS Data and Projects (0000051)\03 - Project Files and Outputs\Access\TOOLBOX\TransitNetworkAnalysisTools_0.5.3\Transit Network Analysis Tools.pyt, in CalculateAccessibilityMatrix:
Line 58:    params = [

File T:\13-1400-10\TPL - GIS Data and Projects (0000051)\03 - Project Files and Outputs\Access\TOOLBOX\TransitNetworkAnalysisTools_0.5.3\Transit Network Analysis Tools.pyt, in CalculateAccessibilityMatrix:
Line 55:    def getParameterInfo(self):

File C:\Program Files\ArcGIS\Pro\Resources\ArcPy\arcpy\geoprocessing\_base.py, in <lambda>:
Line 511:   return lambda *args: val(*gp_fixargs(args, True))

ExecuteError: OD Cost Matrix script failed.
Failed to execute (CalculateAccessibilityMatrix).

See below screenshot for more detail.

![Capture](https://user-images.githubusercontent.com/106562460/197008790-43d637c9-e35a-440c-8466-30ee57bc5508.PNG)

**Comment by mmorang** [comment]
Hmm, it's hard to say from this error what's really going on.  However, I noticed that you appear to be using version 0.5.3 for the Transit Network Analysis Tools toolbox, which is pretty old by this point.  The current version is 0.7.1.

Download the latest version from [ArcGIS Online](https://www.arcgis.com/home/item.html?id=23cc1b720eb0418db4d0e61955b8fe43) or by cloning this GitHub repo, and try again with the updated version.  Maybe the problem will just go away.

**Comment by mmorang** [comment]
Closing this since I haven't heard back from the user.  I'm assuming either the problem is resolved or is no longer important.

**Comment by vancouvertransportation** [comment]
Yes, sorry I haven’t replied Melinda. I updated the toolbox and it started working properly without error. Thanks again for all your help!! ☺

Simon

Simon Wong (he/him)
Planner I | Community Transportation | City of Vancouver
phone: (604) 326-4830, or ext. 84830 | e-mail: ***@***.***

From: Melinda Morang ***@***.***>
Sent: Wednesday, June 7, 2023 8:05 AM
To: Esri/public-transit-tools ***@***.***>
Cc: Wong, Simon ***@***.***>; Author ***@***.***>
Subject: [EXT] Re: [Esri/public-transit-tools] Getting ExecuteError when trying to loop CalculateAccessibilityMatrix (Issue #169)

City of Vancouver security warning: Do not click on links or open attachments unless you were expecting the email and know the content is safe.
________________________________

Closing this since I haven't heard back from the user. I'm assuming either the problem is resolved or is no longer important.
